docs/node-operators/staking-and-snarking.mdx

```diff
@@ -152,7 +186,7 @@ args={["-run-snark-worker $MINA_PUBLIC_KEY", "-snark-worker-fee <fee>"]}
 
 As a SNARK-worker, you get to share some of the block rewards for each block that include your compressed transactions. The block producer is responsible for gathering compressed transactions before including them into a block and is incentivized by the protocol to reward SNARK-workers.
 
-SNARK-workers can be fairly compute intensive. If you need to limit their CPU usage, specify the number of threads that SNARK-workers use with the `-snark-worker-parallelism` flag. This can be especially useful if you're trying to run a block producer and SNARK-worker on the same machine and having issues producing blocks in time.
+SNARK-workers can be fairly compute intensive. If you need to limit their CPU usage, specify the number of threads that SNARK-workers use with the `-snark-worker-parallelism` flag when running a `mina daemon` as a SNARK-worker. This can be especially useful if you're trying to run a block producer and SNARK-worker on the same machine and having issues producing blocks in time.
```

Comment:
```suggestion
SNARK workers can be fairly compute-intensive. If you need to limit their CPU usage, specify the number of threads that SNARK workers use with the `-snark-worker-parallelism` flag when running a `mina daemon` as a SNARK worker. This can be especially useful if you're trying to run a block producer and SNARK worker on the same machine and are having issues producing blocks in time.
```